app/controllers/visit_requests_controller.rb

```diff
@@ -32,4 +32,8 @@ def event
   def visit_request
     @visit_request ||= VisitRequest.find(params[:id])
   end
+
+  def user_verification_status
+    current_user.verified? ? :success_for_verified : :success_for_newbies
```

Comment:
Have no time now to code. Let's merge and discuss this problem personally latter